client/src/commands/encointer_ceremonies.rs

```diff
@@ -772,7 +909,12 @@ async fn get_attendees_for_community_ceremony(
 	let mut attendees = Vec::new();
 	let mut noshows = Vec::new();
 	for storage_key in storage_keys.iter() {
-		match api.get_storage_by_key(storage_key.clone(), at_block).await.unwrap().unwrap() {
+		match api
+			.get_storage_by_key(storage_key.clone(), at_block)
+			.await
+			.unwrap_or(Some(Reputation::VerifiedLinked(0)))
+			.unwrap()
+		{
```

Comment:
I'm sceptical about this workaround as it could fire back. but it works temporarily to support pre-migration type